src/index.html

```diff
@@ -6,7 +6,42 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__month calendar__month-length--29">
+        <div class="calendar__day calendar__day--1 calendar__day-start--sun"></div>
```

Comment:
use nth-child instead one more class
Also, you can do the same logic with 1 wrapper
```suggestion
      <div class="calendar__month calendar__month-length--29 calendar__day-start--sun">
        <div class="calendar__day"></div>
```